cmd/svcat/binding/unbind_cmd.go

```diff
@@ -83,6 +107,41 @@ func (c *unbindCmd) Run() error {
 	var bindings []types.NamespacedName
 	var err error
 
+	if c.abandon {
+		fmt.Fprintln(c.Output, "This action is not reversible and may cause you to be charged for the broker resources that are abandoned.")
+		if !c.skipPrompt {
+			fmt.Fprintln(c.Output, "Are you sure? [y|N]: ")
+			s := bufio.NewScanner(c.stdin)
+			s.Scan()
+
+			err = s.Err()
+			fmt.Fprintln(c.Output, err)
+
+			if strings.ToLower(s.Text()) != "y" {
+				err = fmt.Errorf("aborted abandon operation")
+				return err
+			}
+		}
+
+		if c.instanceName == "" {
+			err = fmt.Errorf("Please provide instance name; aborted abandon operation")
+			return err
+		}
+
+		si := &v1beta1.ServiceInstance{ObjectMeta: metav1.ObjectMeta{Name: c.instanceName, Namespace: c.Namespace}}
+		instanceBindings, err := c.App.RetrieveBindingsByInstance(si)
+		if err != nil {
+			return err
+		}
+		for _, binding := range instanceBindings {
```

Comment:
I don't think it's a good idea to reach in and do this sort of stuff in the command layer. I think I would add some sort of RemoveFinalizers() method to the interface in pkg/svcat/service-catalog, and implement this there, like the rest of the methods like RetrieveBindingsByInstance or Unbind